examples/global-boundaries.ipynb

```diff
@@ -0,0 +1,648 @@
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gdf.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa2a8b2b-53c5-4f22-8b6e-758bbb55e83f",
+   "metadata": {},
+   "source": [
+    "## Preparing the data for our Map\n",
+    "\n",
+    "In this demonstration, we want to end up with a pickable map of municipal boundaries in the Americas with nice looking borders around municipal areas so that we can see the boundaries, regardless of whether they're being activately picked.\n",
+    "To accomplish that, we'll create a lonboard `PathLayer` along with our `SolidPolygonLayer`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0bd5411-d881-4d02-85d4-3c63340a7dad",
+   "metadata": {},
+   "source": [
+    "Our dataset contains `Polygon` and `MultiPolygon` geometries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "7b31fb99-b594-495e-aa3f-437646dfe470",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>shapeName</th>\n",
+       "      <th>shapeID</th>\n",
+       "      <th>geometry</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Mississippi</td>\n",
+       "      <td>66186276B69138566591314</td>\n",
+       "      <td>MULTIPOLYGON (((-88.77009 30.24290, -88.75279 ...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     shapeName                  shapeID  \\\n",
+       "0  Mississippi  66186276B69138566591314   \n",
+       "\n",
+       "                                            geometry  \n",
+       "0  MULTIPOLYGON (((-88.77009 30.24290, -88.75279 ...  "
+      ]
+     },
+     "execution_count": 67,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mississippi = gdf[gdf['shapeName'] == 'Mississippi']\n",
+    "mississippi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a899d398-ecb2-4557-b803-71e0c0a42470",
+   "metadata": {},
+   "source": [
+    "For MultiPolygons, we can get the `unary_union` of the polygon to obtain a `boundary` of the `MultiPolygon`"
```

Comment:
Here we introduce the idea of `unary_union`, but then it's never used below when constructing the boundary. It might be easier to follow if we removed this `unary_union` example. What do you think?

Reply:
Sounds good. Removed via https://github.com/developmentseed/lonboard/pull/351#discussion_r1503471224